test/functional/feature_blockfilterindex_prune.py

```diff
@@ -51,17 +52,17 @@ def run_test(self):
         self.log.info("prune below the blockfilterindexes best block while blockfilters are disabled")
         pruneheight_new = self.nodes[0].pruneblockchain(1000)
         assert_greater_than(pruneheight_new, pruneheight)
-        self.stop_node(0, expected_stderr='Warning: You are starting with governance validation disabled. This is expected because you are running a pruned node.')
+        self.stop_node(0, expected_stderr=EXPECTED_STDERR_NO_GOV_PRUNE)
 
         self.log.info("make sure we get an init error when starting the node again with block filters")
         self.nodes[0].assert_start_raises_init_error(
             extra_args=["-fastprune", "-prune=1", "-blockfilterindex=1"],
-            expected_msg="Warning: You are starting with governance validation disabled. This is expected because you are running a pruned node.\nError: basic block filter index best block of the index goes beyond pruned data. Please disable the index or reindex (which will download the whole blockchain again)",
+            expected_msg=EXPECTED_STDERR_NO_GOV_PRUNE + "\nError: basic block filter index best block of the index goes beyond pruned data. Please disable the index or reindex (which will download the whole blockchain again)",
```

Comment:
Nit: you can use here f-string to unify with other code:

expected_msg=f"{EXPECTED_STDERR_NO_GOV_PRUNE}\nError: basic block filter index best block of the index goes beyond pruned data. Please disable the index or reindex (which will download the whole blockchain again)",

Reply:
Resolved in latest push